.github/CODEOWNERS

```diff
@@ -26,3 +26,14 @@ x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/sto
 # APM Data index templates, etc.
 x-pack/plugin/apm-data/src/main/resources @elastic/apm-server
 x-pack/plugin/apm-data/src/yamlRestTest/resources @elastic/apm-server
+
+# Delivery
+gradle @elastic/es-delivery
+build-conventions @elastic/es-delivery
+build-tools @elastic/es-delivery
+build-tools-internal @elastic/es-delivery
+*.gradle @elastic/es-delivery
+.buildkite @elastic/es-delivery
+.ci @elastic/es-delivery
+.idea @elastic/es-delivery
+distribution @elastic/es-delivery
```

Comment:
This is going to include the stuff under `tools` which is primarily owned by core/infra. I think if we just include `src`, `docker/src` and `packages/src` that should capture what we want given everything else in there is build scripts which is already captured by the `*.gradle` rule.